src/parser/core/modules/Procs.tsx

```diff
@@ -312,6 +312,16 @@ export abstract class Procs extends Analyser {
 	*/
 	protected jobSpecificOnConsumeProc(_procGroup: ProcGroup, _event: Events['action']): void { /* */ }
 
+	/**
+	 * May be overridden by subclasses. Called by onCast to determine whether the action in question can consume
+	 * multiple procs from a single event.
+	 * @param _action The action that might consume multiple procs
+	 * @returns False by default. Jobs may override to return true, allowing usages to be registered for more than one proc status from the event
+	 */
+	protected checkMayConsumeAdditionalProcs(_action: number): boolean {
```

Comment:
_Consider:_ This might be a better name if you drop the `check`. `mayConsumeAdditionalProcs()` (or even `actionMayConsumeAdditionalProcs()`) seems a clearer indicator of "this returns a boolean" to me.

Reply:
I like actionMay... we'll go with that.